src/main/java/fact/starservice/Star.java

```diff
@@ -0,0 +1,22 @@
+package fact.starservice;
+
+import fact.coordinates.EquatorialCoordinate;
+
+public class Star implements  Comparable<Star> {
```

Comment:
Could you add a docstring about the class?

Reply:
done